.github/workflows/drc_io_sram_test.yml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2022 GlobalFoundries PDK Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is a Github Actions for testing DRC rule deck of GF180MCU
+name: DRC IO and SRAM test
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or closed pull request events
+  push:
+  pull_request:
+    types:
+      - closed
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# Defines environment variables
+env:
+  PDK_ROOT: /home/runner/work/globalfoundries-pdk-libs-gf180mcu_fd_pr/globalfoundries-pdk-libs-gf180mcu_fd_pr/rules/klayout
```

Comment:
I think this should be `${{ env.GITHUB_WORKSPACE }}` -- see https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables

Reply:
I will change it from
```
PDK_ROOT: /home/runner/work/globalfoundries-pdk-libs-gf180mcu_fd_pr/globalfoundries-pdk-libs-gf180mcu_fd_pr/rules/klayout
```
to
```
PDK_ROOT: ${{ env.GITHUB_WORKSPACE }}/rules/klayout
```